pkgs/misc/emulators/proton-caller/default.nix

```diff
@@ -1,31 +1,25 @@
-{ lib, fetchFromGitHub, rustPlatform, installShellFiles }:
+{ lib, fetchFromGitHub, rustPlatform }:
 
 rustPlatform.buildRustPackage rec {
   pname = "proton-caller";
-  version = "2.2.3";
+  version = "2.3.0";
 
   src = fetchFromGitHub {
     owner = "caverym";
     repo = pname;
     rev = version;
-    sha256 = "0968pmapg6157q4rvfp690l1sjnws8hm62lvm8kaaqysac339z7z";
+    sha256 = "1rj0f8jzmrvj6gz1rcdjmxdqk2i5cxhz9ji4217kwyb6h1h0jmdk";
   };
 
-  cargoSha256 = "1vp2vvgy8z350a59k1c3s5ww6w2wikiha4s7jkkz9khl0spn19a8";
+  cargoSha256 = "165kzza1m8h37y1ir0d0hp0z645h4ihkyj83fii69f18gk47r3kg";
 
-  nativeBuildInputs = [ installShellFiles ];
-
-  outputs = [ "out" "man" ];
-
-  postInstall = ''
-    installManPage manual/proton-call.6
-  '';
+  outputs = [ "out" ];
```

Comment:
Where did the manpage go?

Reply:
There isn't a man page anymore for some reason.